src/main/java/net/minecraftforge/client/event/RegisterClientCommandsEvent.java

```diff
@@ -53,4 +56,12 @@ public CommandDispatcher<CommandSourceStack> getDispatcher()
     {
         return dispatcher;
     }
+
+    /**
+     * @return the context to build the commands for
```

Comment:
This should use an inline `{@return}` tag:
```suggestion
     * {@return the context to build the commands for}
```